schema/deploy/tables/funding_stream_rfp_project_state.sql

```diff
@@ -0,0 +1,46 @@
+-- Deploy cif:tables/funding_stream_rfp_project_state to pg
+-- requires: tables/funding_stream_rfp
+-- requires: tables/project_state
+
+BEGIN;
+
+create table cif.funding_stream_rfp_project_state (
+  id integer primary key generated always as identity,
```

Comment:
alternatively, you could make a compound (funding_stream_rfp_id, project_state_id) primary key

Reply:
I think its easier to use an id here for when you're selecting a project_state_id on a project.